src/Billing/Controllers/FreshdeskController.cs

```diff
@@ -77,7 +77,9 @@ public class FreshdeskController : Controller
 
                 foreach (var org in orgs)
                 {
-                    var orgNote = $"{org.Name} ({org.Seats.GetValueOrDefault()}): " +
+                    // Prevent org names from injecting any additional HTML
+                    var orgName = org.Name.Replace("<", string.Empty).Replace(">", string.Empty);
+                    var orgNote = $"{orgName} ({org.Seats.GetValueOrDefault()}): " +
```

Comment:
We are considering Password Manager Seat only? because org.Seats.GetValueOrDefault() is for password maanger seat only while org.SmSeats.GetValueOrDefault() is for secrets Manager Seat

Reply:
I don't see org name being included anywhere else.

Reply:
IMO updating the content of the notes field is out of scope here. The changes address the stated issue of preventing an HTML injection attack. I'll resolve this and create a spike to investigate if we should add `SmSeats` to the FD note.